### DISCUSSION TITLE
Print to stderr shows up in notebook

### DISCUSSION THREAD

**Discussion by fonsp** [JuliaPluto/PlutoTest.jl#27] [Issues]
Since https://github.com/fonsp/Pluto.jl/pull/1957, the fallback printing also shows up in Pluto :(

![image](https://user-images.githubusercontent.com/6933510/157517449-55cdc4cd-8d21-458f-adc9-171586aa09fe.png)

@dralletje Why was this added? Just debugging?

https://github.com/JuliaPluto/PlutoTest.jl/blob/299765f6697b1cd1ab470f00dc5c4180ab406ac3/src/notebook.jl#L1684-L1686

**Comment by dralletje** [comment]
Oof. It was for compatibility with `] test`, didn't think it would hurt if we'd show it in Pluto 🤦‍♂️ so I'd still keep it, but "just" not when we're in Pluto..
